src/AlphaVantage/CryptoDataHandlers.js

```diff
@@ -1,15 +1,15 @@
 import apiKey from "./AlphaVantageKey";
 import digitalCurrencyList from "./DigitalCurrencyList";
 
-const fetchCryptoSymbols = (searchQuery) => {
+export const fetchCryptoSymbols = (searchQuery) => {
```

Comment:
using `export` directly on variable constants is usually easier to read and understand if the function is exported or not, rather than defining the export list at the bottom of the file.